src/coredump/coredumpctl.c

```diff
@@ -1150,6 +1196,24 @@ static int run_debug(int argc, char **argv, void *userdata) {
         if (r < 0)
                 return log_oom();
 
+        if (arg_root) {
+                if (streq(arg_debugger, "gdb")) {
+                        const char *sysroot_cmd;
+                        sysroot_cmd = strjoina("set sysroot ", arg_root);
```

Comment:
should this quote/escape the path? i am not sure what gdb expects precisely, but if we have a path that includes a quote and a space, would this even work?

there are escaping helpers already, i.e. cescape() and so on.

Reply:
I tested with a path containing a quote and a space and it works without escaping.